admin/helpers/school-home-pin-generation-eligibility-presenter.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const moment = require('moment')
+
+const checkWindowV2Service = require('../services/check-window-v2.service')
+const config = require('../config')
+const monitor = require('../helpers/monitor')
+
+const schoolHomePinGenerationEligibilityPresenter = {}
+
+/**
+ * Fetch data for familiarisation and live pin generation eligibility
+ * @returns {Object} Eligibility data including flags and relevant datetimes
+ */
+schoolHomePinGenerationEligibilityPresenter.getPresentationData = async () => {
+  const checkWindow = await checkWindowV2Service.getActiveCheckWindow()
```

Comment:
What happens if there isn't an active check window?

Reply:
@jon-shipley we shouldn't reach that case ever since the expectation would be for the teacher to be unable to login on the teacher admin panel when there is no active check window